inc/core/patterns/bcgov-2x2-picture-and-text-grid.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * 2x2 Picture and Text Grid.
+ *
+ * @since 1.7.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+
+ namespace Bcgov\Theme\Block;
+
+return [
+	'title'      => __( '2x2 Picture and Text Grid', 'bcgov_blocks_theme' ),
+	'categories' => [ 'bcgov-blocks-theme-general' ],
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0","bottom":"0"}}},"backgroundColor":"white","layout":{"type":"constrained"}} -->
+	<div class="wp-block-group alignfull has-white-background-color has-background" style="margin-top:0;margin-bottom:0;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:media-text {"align":"full","mediaId":4557,"mediaLink":"none","mediaType":"image","imageFill":false,"style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}},"backgroundColor":"custom-color-1","className":"is-style-default"} -->
+	<div class="wp-block-media-text alignfull is-stacked-on-mobile is-style-default has-custom-color-1-background-color has-background" style="margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><figure class="wp-block-media-text__media"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/default_04.jpg" alt="" class="wp-image-4557 size-full"/></figure><div class="wp-block-media-text__content"><!-- wp:group {"style":{"spacing":{"margin":{"top":"4rem","bottom":"4rem"},"padding":{"top":"0","right":"2rem","bottom":"0","left":"2rem"}}},"backgroundColor":"custom-color-1","layout":{"type":"constrained","justifyContent":"left","contentSize":"642px"}} -->
```

Comment:
We may want to find images that are more representative of government sites as defaults. The fruit and veg defaults from the original patterns are hard for people to wrap their heads around. It might be worthwhile having a look through the StrongerBC content to see if any of the cards on that site could be useful.

Reply:
Sure, I can grab some images from that site and replace the fruit/veg ones. In some cases they might be ok, but there are definitely instances where they're not ideal.